pages/1.10/security/ent/tls-ssl/ca-custom/index.md

```diff
@@ -8,26 +8,322 @@ excerpt:
 enterprise: true
 ---
 
-Each DC/OS cluster has its own DC/OS certificate authority (CA). By default, that CA uses an automatically generated globally unique root CA certificate for signing other certificates. In lieu of using the auto-generated root CA certificate, you can configure DC/OS Enterprise to use a custom CA certificate, which can be a root CA certificate or an intermediate CA certificate.
+# Motivation
+Each DC/OS Enterprise cluster has its own DC/OS certificate authority (CA). By default, that CA uses a globally unique root CA certificate generated during the installation of DC/OS. That root CA certificate is used for signing certificates for the components of DC/OS, such as Admin Router. In lieu of using the auto-generated root CA certificate, you can configure DC/OS Enterprise to use a custom CA certificate, which is *either* a root CA certificate *or* an intermediate CA certificate. (see examples [below](#example-use-cases))
 
 The benefits of using a custom CA certificate for your DC/OS Enterprise cluster include:
 
 - Ensuring that all X.509 certificates used within the DC/OS cluster (for both signing and encrypting) derive from your organization's X.509 certification hierarchy.
 - Controlling security properties of the key pair (such as type and strength) used for signing DC/OS component certificates.
 - Ensuring that all DC/OS components (including Admin Router) present browser-trusted certificates.
 
-Custom CA certificate support is enabled by three install time [configuration parameters](/1.10/installing/ent/custom/configuration/configuration-parameters/#ca-certificate-path-enterprise):
+# The structure of this document page
+To facilitate the reading of this page we start out by providing a glossary for general definition of terms, followed by an in-depth configuration parameter reference. An installation walkthrough is provided in section [Installing DC/OS Enterprise with a custom CA certificate](#installing-dcos-enterprise-with-a-custom-ca-certificate). Section [Example use cases](#example-use-cases) then provides example file contents for the custom CA certificate configuration files for three popular use cases.
 
-- `ca_certificate_path`
-- `ca_certificate_key_path`
-- `ca_certificate_chain_path`
+# What is supported and what is not
+- Only custom CA certificates that have an associated RSA-type key pair are supported. Other types of certificates, such as those using ECC-type key pair, are currently not supported. Support for ECC-type key pairs will be added in the future.
+- You have to use a custom CA certificate only with a fresh installation of DC/OS Enterprise 1.10 or higher. Older versions of DC/OS are not supported.
```

Comment:
Does the use fo the word "fresh" mean that it is not possible to perform these actions during an upgrade? Perhaps add this explicitly:

- Custom CA certificates are only supported for a fresh installation of DC/OS Enterprise 1.10 or higher. Older versions of DC/OS are not supported, and it is not possible to add a custom CA certificate during an upgrade.

Reply:
@jgehrcke @mhrabovcin could you please confirm that using custom CA is not supported by the DC/OS upgrade? I haven't tried updating DC/OS so far. 
The existing doc does not mention the upgrade explicitly: https://docs.mesosphere.com/1.10/security/ent/tls-ssl/ca-custom/

Reply:
@ahairape yes, this is the case. You can find this information in the design document. Please follow @jongiddy's suggestion, I like it very much.